sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -574,7 +574,7 @@ class PlanResolutionSuite extends AnalysisTest {
         assert(ctas.catalog.name == "testcat")
         assert(ctas.tableName == Identifier.of(Array("mydb"), "table_name"))
         assert(ctas.properties == expectedProperties)
-        assert(ctas.writeOptions == Map("other" -> "20"))
+        assert(ctas.writeOptions == Map.empty)
```

Comment:
Should we add a test where this is non empty?

Reply:
The SQL CTAS can't specify write options so it's always empty. It's kind of a general problem in the SQL API, as we can't specify options for SELECT/INSERT while `DataFrameWriterV2` can.

Reply:
I agree. We can add tests if/when we add an OPTIONS clause to CTAS.